### PR TITLE
Update pontifical-gregorian-university.csl

### PR DESCRIPTION
**Pull request by PUG-IS**
Change one macro.
Add redefined localization data.
Shortly i'll create dependent styles.

### CODE REVIEW

File: pontifical-gregorian-university.csl [hunk 1]
```diff
@@ -12,22 +12,69 @@
...
     <terms>
       <term name="et-al"> &#8211; al.</term>
       <term name="ibid">Ibid.</term>
-      <term name="accessed">accesso</term>
+      <term name="in">in</term>
+      <term name="accessed">accesso:</term>
+      <term name="editor">
+        <single>ed.,</single>
+        <multiple>ed.,</multiple>
+      </term>
```
Comment (rmzelle) [inline pontifical-gregorian-university.csl]: > `<single>ed.,</single>`

It's generally better to keep separating punctuation (which I assume this comma is) out of the term definitions, and use a prefix/suffix or (even better yet) a delimiter instead.

**Comment by PUG-IS** [comment]
Done.

File: pontifical-gregorian-university.csl [hunk 2]
```diff
@@ -12,22 +12,69 @@
...
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale>
     <style-options punctuation-in-quote="false"/>
     <terms>
       <term name="et-al"> &#8211; al.</term>
       <term name="ibid">Ibid.</term>
-      <term name="accessed">accesso</term>
+      <term name="in">in</term>
+      <term name="accessed">accesso:</term>
```
Comment (rmzelle) [inline pontifical-gregorian-university.csl]: Same here, by the way. (colons are almost always delimiters)

**Comment by rmzelle** [comment]
> Done.

Are you sure? I saw you removed all the commas in the term definitions (https://github.com/citation-style-language/styles/pull/2098/commits/44da57c7f271e0ac5b83f4fa54ff44f6cdfcb5e0), but you didn't put in any replacements as prefixes/suffixes/delimiters. Is that correct?

**Comment by rmzelle** [comment]
Much better :). Let me know if this is ready to be merged.

**Comment by PUG-IS** [comment]
> Same here, by the way. (colons are almost always delimiters)

You are right...thanks..
I just corrected the style...

>Are you sure? I saw you removed all the commas in the term definitions (44da57c), but you didn't put in any replacements as prefixes/suffixes/delimiters. Is that correct?

Yes..I had already used
`<label form="short" prefix=", "/>`
into the macros.
Thanks again...

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**